rmw_fastrtps_shared_cpp/src/rmw_client.cpp

```diff
@@ -126,4 +126,18 @@ __rmw_destroy_client(
   RCUTILS_CAN_RETURN_WITH_ERROR_OF(RMW_RET_ERROR);  // on completion
   return final_ret;
 }
+
+rmw_ret_t
+__rmw_client_get_actual_qos(
+  const rmw_client_t * client,
+  rmw_qos_profile_t * qos)
+{
+  auto info = static_cast<CustomClientInfo *>(client->data);
+
+  eprosima::fastdds::dds::DataReader * fastdds_dr = info->response_reader_;
```

Comment:
retrieve actual qos from data reader on client, this is assuming that data writer(request_writer_) possesses the same actual qos? probably it would be nice to check? and if they do not match, it prints the warning message?

Reply:
Done in 7004b814c49b4b118fd2894e54ca69b9e37d27bb